.vscode/cspell.json

```diff
@@ -81,7 +80,7 @@
     "sdk/servicefabric/azure-servicefabric/**",
     "sdk/search/azure-search-documents/**",
     "sdk/storage/azure-storage-blob-changefeed/**",
-    "sdk/servicebus/azure-servicebus/**",
+    
```

Comment:
good catch! We don't want to ignore all of servicebus, but we can ignore the test files when spell checking since those are just for internal testing.
Nit:
```suggestion
```

After line 33, we can add the following:
```
    {
       "filename": "sdk/servicebus/azure-servicebus/tests/mgmt_tests/*.py",
       "words": [
           "cpqmva",
           "fjruid",
           "rekocd",
           "eqkovcd",
           "eqkovc",
           "fjrui",
           "dfjdfj",
           "kkaqo",
           "dfjfj",
           "sagho",
           "testaddf",
           "iewdm",
           "rtofdk",
           "dqkodq"
       ]
    },
```